sql/job_api.sql

```diff
@@ -7,7 +7,8 @@ CREATE OR REPLACE FUNCTION add_job(
   schedule_interval INTERVAL,
   config JSONB DEFAULT NULL,
   initial_start TIMESTAMPTZ DEFAULT NULL,
-  scheduled BOOL DEFAULT true
+  scheduled BOOL DEFAULT true,
+  check_config REGPROC DEFAULT NULL
```

Comment:
I am not convinced having one additional function for checking is the right approach. It also seems that the check function is optional, which I am not sure is a good idea.

Why not simply use the already defined procedure also for validation?